data/monitoring/prom-rule.yaml

```
@@ -43,8 +45,10 @@ spec:
             severity: warning
             kubernetes_operator_part_of: kubevirt
             kubernetes_operator_component: cluster-network-addons-operator
+        # +help:summary="Total count of running KubeMacPool manager pods",type=Gauge
         - expr: sum(up{namespace='{{ .Namespace }}', pod=~'kubemacpool-mac-controller-manager-.*'} or vector(0))
           record: kubevirt_cnao_kubemacpool_manager_num_up_pods_total
+        # +help:summary="Total count of deployed KubeMacPool CRs",type=Gauge
```

This is more accurate
```suggestion
        # +help:summary="Total count of KubeMacPool manager pods deployed by CNAO CR",type=Gauge
```

---

updated